src/views/StoriesList.vue

```diff
@@ -89,7 +89,7 @@ function closeSnackBar() {
       <v-row align="center" class="mb-4">
         <v-col cols="10"
           ><v-card-title class="pl-0 text-h4 font-weight-bold"
-            >Recipes
+            >Stories
```

Comment:
why is the greater than signs on the line after the matching less than sign?

Reply:
I guess I didn't have the Vue formatter configured in vscode. Thanks,